osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -22,6 +23,8 @@ public class DrawableSpinner : DrawableOsuHitObject
         private readonly Container circleContainer;
         private readonly DrawableHitCircle circle;
 
+        private float beatmapOd;

```

Comment:
Overall Difficulty is frequently referred to as OD, should probably be maintained here too. (would require adding it as an abbreviation to the solution's DotSettings)

The ```//TODO``` you removed below also referred to it as OD.